acbuild/copy.go

```diff
@@ -20,9 +20,9 @@ import (
 
 var (
 	cmdCopy = &cobra.Command{
-		Use:     "copy PATH_ON_HOST PATH_IN_ACI",
+		Use:     "copy PATHS_ON_HOST PATH_IN_ACI",
 		Short:   "Copy a file or directory into an ACI",
-		Example: "acbuild copy nginx.conf /etc/nginx/nginx.conf",
+		Example: "acbuild copy stuff/* nginx.conf /etc/nginx/",
```

Comment:
Cobra only lets me set a single example. I'll put examples for both in the documentation, but which would you prefer goes here?